Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -604,32 +607,16 @@ PhysicalParticleContainer::AddPlasmaFromFile(ParticleReal q_tot,
         double momentum_unit_y = 1.0;
         if (ps["momentum"].contains("y")) {
             ptr_uy = ps["momentum"]["y"].loadChunk<ParticleReal>();
-             momentum_unit_y = ps["momentum"]["y"].unitSI();
+            momentum_unit_y = ps["momentum"]["y"].unitSI();
         }
         series->flush();  // shared_ptr data can be read now
 
-        ParticleReal weight = 1.0_prt;  // base standard: no info means "real" particles
-        if (q_tot != 0.0) {
```

Comment:
If we remove this part (for now), then we also need to update the documentation please.

Docs:
- https://warpx.readthedocs.io/en/latest/usage/parameters.html#particle-initialization
- File to change: `Docs/source/usage/parameters.rst`

Reply:
Let's also issue a warning if `q_tot` is found in he inputs file and ignored.

Reply:
done :+1: